modules/core/include/opencv2/core/cuda.inl.hpp

```diff
@@ -343,6 +343,12 @@ bool GpuMat::empty() const
     return data == 0;
 }
 
+inline
+void* GpuMat::cudaPtr() const
+{
+    return data;
```

Comment:
There is `CV_PROP_RW` macro that allows to expose object properties to Python and other languages. You do not need own method for it.

Reply:
I believe current approach is fine.
We should not allow changing of this pointer through `CV_PROP_RW`.